src/axom/spin/docs/sphinx/bvh.rst

```diff
@@ -84,3 +85,44 @@ is set to the new surface element. The predicate used for traversal also utilize
 current-minimum candidate data to avoid traversing internal nodes that are farther than
 the current minimum squared distance.
 
+Example: Broad-phase collision detection
+========================================
+
+Below is an example of how the device traversal object might be used in a broad-
+phase collision detection problem. First, we initialize the BVH with the bounding
+boxes of all the query objects, and create a traverser object:
```

Comment:
I think the example program is testing each element in a surface mesh against each other element, to see if any elements intersect.  If my understanding is correct, will you put in the following change:
```suggestion
The following example tests each element of a surface mesh for intersection with
each other element. It provides an example of how the device traversal object
might be used in a broad-phase collision detection problem. First, we initialize
the BVH with the bounding boxes of all the query objects (mesh elements), and
create a traverser object:
```